### ISSUE TITLE
A better way to deal with the Peertube links

### ISSUE THREAD

**Issue by tio-trom** [friendica/friendica#14050]
If I share a Peertube link on Friendica it looks like this (https://social.trom.tf/display/dbc8dc44-33efc893-2c6008433e7c597f):

![image](https://github.com/friendica/friendica/assets/38043315/25afea4d-4d93-49f0-b934-87a03eafbc88)

Not bad, but since it lacks a thumbnail and it also showcases the name of the profile and not the name of the channel, it does look a bit off. Tio is not the name of that channel. This is the channel https://videos.trom.tf/c/trom/videos and it is one of the many channels I have. 

It is also creating a second "preview" at the bottom repeating the same information:

![2024-03-28_15-51](https://github.com/friendica/friendica/assets/38043315/ba98ca6a-0008-4757-83b2-d4e164b85364)

It is also not convenient that when you play the video from Friendica it basically plays an mp4 file and you can't even skip forward, you have to wait for it to load and load. It is not a good experience at all.

Worse, is that when someone sees that post I just made on Friendica, on their Mastodon, it looks like so (https://mastodon.social/@trom@social.trom.tf/112173193563389499):

![2024-03-28_15-52](https://github.com/friendica/friendica/assets/38043315/7db3256c-6e06-4585-b348-4d8cc4de5331)

So is there a way to simply embed a Peertube link with the thumbnail, description and title like you may do with any normal website shared in Friendica? The best would be to embed the Peertube video, but as discussed here it seems difficult - https://github.com/friendica/friendica/issues/10019

Mastodon seems to be able to deal with Peertube much better - see https://mastodon.social/@peertube@framapiaf.org/111577572314609223 

I hope there is a solution for this since we use Peertube a lot together with Friendica :)

**Comment by annando** [comment]
We share Peertube videos like we share other posts from the Fediverse. Mastodon can't handle quoted posts, so it displays them this way.

**Comment by tio-trom** [comment]
hm interesting. Why isn't this the case for me I wonder. When I share a Peertube post it embeds the video itself. If you share say this video https://videos.trom.tf/w/ifXJJxjs1eBcSvHDSUVLM7 on your instance, won't it look like the above? Embed mp4 video?

**Comment by haheute** [comment]
I have also just shared a peertube link ( https://tube.tchncs.de/w/3WKWmruT4CHCsvVdYZKkuV ) and the result was not, what the preview showed. It showed the preview image of the video, but the resulting post had an embedded player with another picture in it.

**Comment by tio-trom** [comment]
> I have also just shared a peertube link ( https://tube.tchncs.de/w/3WKWmruT4CHCsvVdYZKkuV ) and the result was not, what the preview showed. It showed the preview image of the video, but the resulting post had an embedded player with another picture in it.

Yes this is what I also see. The preview looks ok with the title of the video and thumbnail:
![2024-04-02_15-50](https://github.com/friendica/friendica/assets/38043315/2d52c353-d95e-44d3-9e84-b243119a5507)

And then when I click share I see it like so:
![2024-04-02_15-50_1](https://github.com/friendica/friendica/assets/38043315/28b8e1f0-802b-4ff8-a0a9-8af7fec35e92)

Embed...
